doc/nrf/device_guides/working_with_nrf/nrf53/nrf5340.rst

```diff
@@ -598,9 +598,9 @@ If not changed, then the default child image prefix for MCUboot is ``mcuboot_``
 
 .. note::
 
-   The application core can be reverted, but doing so bricks the network core upon revertion, as the reversion process fills the network core with the content currently in the RAM that pcd uses.
-   To enable this, define the ``USE_NRF53_MULTI_IMAGE_WITHOUT_UPGRADE_ONLY`` Kconfig option in the project-level Kconfig file.
-   When this is option is defined, you can enable it by setting :kconfig:option`CONFIG_USE_NRF53_MULTI_IMAGE_WITHOUT_UPGRADE_ONLY`.
+   The application core can be reverted, but doing so bricks the network core upon reversion, as the reversion process fills the network core with the content currently in the RAM that pcd uses.
```

Comment:
```suggestion
   The application core can be reverted, but doing so bricks the network core upon reversion, as the reversion process fills the network core with the content currently in the RAM that PCD uses.
```
Not so sure about the verb "brick" in this context. Could it be just break?
Should we add PCD to the glossary? Peripheral CPU DFU according to the Kconfig reference.

Reply:
@sigvartmh, could you look at the above comment and suggest?

Reply:
1. I think we should add PCD into the glossary
2. Depends on what your definition is. At this point the application running on the network core won't be bootable as it would be garbage grabbed from RAM. It could be restored with a secondary DFU but that DFU can't be over-the-air as the network core will be non-functional/bricked/broken. So I want it to be clear that the network core cannot be used as there is no run-able application on the network core after a revert.
@peknis 

```
The application core can be reverted, but doing so breaks the network core upon
reversal, as the reversion process fills the network core with the content currently
in the RAM that PCD uses.
```

I'm fine with having it written above if you think that break makes sense.

Reply:
Adding the above version. Glossary update will be done in a separate PR by @kittydepa.

Reply:
PR for adding glossary entry is here: https://github.com/nrfconnect/sdk-nrf/pull/14283